Exporter/src/main/resources/application.yml

```diff
@@ -185,6 +185,7 @@ spring:
 jobs:
   backend-url: http://biostudy-prod.ebi.ac.uk:10080/biostd-prod
   libFileStudies:
+    - NON_EXISTING
```

Comment:
@jhoanmanuelms this is the workaround I apply to fix lib studies

Reply:
Well... C'est la vie mon amie!